build/android/merge_jars.py

```diff
@@ -25,20 +29,38 @@
 from util import build_utils
 
 
+def IsMergeableJar(jar_name):
+  """
+  Returns True if a certain JAR does not have any classes outside the
+  org.chromium and org.xwalk namespaces.
+  """
+  with zipfile.ZipFile(jar_name) as zip_file:
+    for entry_name in zip_file.namelist():
+      if entry_name in ('META-INF/', 'META-INF/MANIFEST.MF') or \
+         fnmatch.fnmatchcase(entry_name, 'SevenZip/*.class') or \
```

Comment:
> outside the org.chromium and org.xwalk namespaces.

This comment does not refer to  other namespaces  `SevenZip`.
Could we put `SevenZip/*.class`, `org/chromium/*.class` and `org/xwalk/*.class` to a list that we can give a general comment and expand it conveniently?